src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,160 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
```

Comment:
I am confused by the name of this component. Please can you help me understand how it differs from `MediumGridWrapper`?

Reply:
`MediumGridWrapper` now renamed to `GridWrapperMedium` acts as a parent grid wrapper to its children. `MediumContentContainer` doesn't. `MediumContentContainer` is meant to act as a wrapper for pages like the `PhotoGallery`, `ArticlePage` and `MAP` which have their contents centered on the page.

Reply:
I'm a bit confused on the naming conventions as well, I don't think `MediumContentContainer` is a good fit. Maybe something tied in with it being a Single column container and then maybe a comment on where the container is used to make it more clear.